ci-operator/step-registry/baremetalds/devscripts/gather/baremetalds-devscripts-gather-ref.yaml

```diff
@@ -7,5 +7,15 @@ ref:
     requests:
       cpu: 1000m
       memory: 500Mi
+  credentials:
+  - namespace: test-credentials
+    name: packet-metal-assisted
+    mount_path: /var/run/secrets/packet/metal-assisted
+  - namespace: test-credentials
+    name: packet-metal-ipi
+    mount_path: /var/run/secrets/packet/metal-ipi
+  - namespace: test-credentials
+    name: packet-metal-sno
+    mount_path: /var/run/secrets/packet/metal-sno
```

Comment:
It's not clear to me the reasons underlying this choice, can you please clarify?
We already have a secret bucket defined that could be easily used - and moreover already bound to `$CLUSTER_PROFILE` - where the various required secrets could be put (discriminated by name by using `$PACKET_ACCOUNT`). In that way such configuration will not be required, as well as in the other steps (see also below)

Reply:
I can try this once more initially when I tried to mount into `CLUSTER_PROFILE_DIR` it didn't let me because it was a read-only. Maybe the approach was wrong though so let me try this again.